astropy/io/fits/util.py

```diff
@@ -380,14 +380,9 @@ def isfile(f):
 def fileobj_open(filename, mode):
     """
     A wrapper around the `open()` builtin.
-
-    This exists because `open()` returns an `io.BufferedReader` by default.
-    This is bad, because `io.BufferedReader` doesn't support random access,
-    which we need in some cases.  We must call open with buffering=0 to get
-    a raw random-access file reader.
     """
 
-    return open(filename, mode, buffering=0)
+    return open(filename, mode)
```

Comment:
```suggestion
    if sys.platform.startswith('win'):
        return open(filename, mode, buffering=0)
    else:
        return open(filename, mode)
```
Unless / until someone come up with a solution for using buffering on Windows, we probably need to keep it at that.

Reply:
Thank you. Was in the middle of making this change right now. :)